Jetchat/build.gradle

```diff
@@ -61,18 +61,32 @@ subprojects {
     tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
         kotlinOptions {
             // Treat all Kotlin warnings as errors
-            allWarningsAsErrors = true
+            allWarningsAsErrors = false
```

Comment:
Can we turn this back on and suppress the specific bits? Or at least added a `FIXME`

Reply:
You're fast. Done.